pkg/server/tokenrequest/tokenrequest.go

```diff
@@ -233,10 +254,10 @@ var tokenTemplate = template.Must(template.New("tokenTemplate").Parse(
   <code>{{.AccessToken}}</code>
 
   <h2>Log in with this token</h2>
-  <pre>oc login <span class="nowrap">--token={{.AccessToken}}</span> <span class="nowrap">--server={{.PublicMasterURL}}</span></pre>
+  <pre>oc login <span class="nowrap">--token={{.AccessToken}}</span> <span class="nowrap">--server={{.PublicMasterURL}}</span><button role="copy-to-clipboard">Copy</button></pre>
```

Comment:
I don't think `copy-to-clipboard` is a valid role.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles